ecosystem/sep-xxxx.md

```diff
@@ -0,0 +1,182 @@
+## Preamble
+
+```
+SEP: To Be Assigned
```

Comment:
Can you assign the next number, and add the SEP to the ecosystem README under the drafts section?

Could you also add a Version field here and set it to 0.0.1?

Reply:
Absolutely, though the [template](https://github.com/stellar/stellar-protocol/blob/master/sep-template.md) did not have a version field?

Reply:
Thanks for pointing that out, the template appears to be out-dated. Updated here: https://github.com/stellar/stellar-protocol/pull/1098